rpms/ose-azure-acr-image-credential-provider.yml

```diff
@@ -0,0 +1,16 @@
+content:
+  source:
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/cloud-provider-azure.git
```

Comment:
For aws we started using the openshift org, not openshift-priv. Is there a step needed to move to priv?

Reply:
`openshift-priv/cloud-provider-azure` exists

Reply:
no additional configuration required.

Reply:
Awesome! Thanks :)